pkg/operator/workloads/logs.go

```diff
@@ -299,6 +430,21 @@ func extractFromCortexLog(match string, loglevel string, logStr string) (*string
 		return formatHeader3(cutStr), false
 	}
 
+	matches := jsonPrefixRegex.FindStringSubmatch(cutStr)
+	if len(matches) == 2 {
+		indent := len(matches[0]) - 1 // matches to curly or square bracket so remove the last char
+		indentStr := cutStr[:indent]
+		maybeJSON := cutStr[len(indentStr):]
+		jsonBytes := []byte(maybeJSON)
+		var obytes bytes.Buffer
+		err := json.Indent(&obytes, jsonBytes, indentStr, "  ")
```

Comment:
Will this break if e.g. in Python we log a map with a number as a key? If so, we might want to consider handling that. One possible way is to use `yaml` to parse it into and `interface{}`, and then use `s.Obj()` on the `interface{}`. What do you think?